doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -568,8 +562,8 @@ Libraries for networking
 
   * Updated:
 
-      * To return negative :file:`errno.h` errors instead of positive ZCBOR errors.
-      * The CoAP download authentication to no longer depend on the :ref:`CoAP Client library <zephyr:coap_client_interface>`.
+    * To return negative :file:`errno.h` errors instead of positive ZCBOR errors.
+    * The CoAP download authentication to no longer depend on the :ref:`CoAP Client library <zephyr:coap_client_interface>`.
```

Comment:
Line 609 to be updated according to https://github.com/nrfconnect/sdk-nrf/pull/22723.